.github/workflows/add-to-docs-project.yml

```
@@ -6,7 +6,7 @@ on:
     types: [labeled]
 jobs:
   main:
-    if: ${{ github.event.label.name == 'type/docs' }}
+    if: ${{ github.event.label.name == 'type/docs' }} && { github.repository == 'grafana/tempo' }}
```

@jdbaldry does this statement do what we need it to do? (Many thanks to @ie-pham for the help.)

---

This looks odd, I'd expect something like:

```suggestion
    if: ${{ github.event.label.name == 'type/docs' &&  github.repository == 'grafana/tempo' }}
```

---

Note that this restricts the workflow to only run on branches in the repository and means that you can't add PRs from forks to the project board (this was already true of the last iteration of this workflow though).

In Slack I planned to replace this workflow with something centralized but I ended up being off all week sick: https://raintank-corp.slack.com/archives/C5PG2JK8W/p1727194434073929